README.md

```diff
@@ -577,6 +579,13 @@ $ php vendor/bin/phpunit
 The test suite contains both unit tests and functional integration tests.
 The functional tests require access to a running Redis server instance
 and will be skipped by default.
+
+For quick execution, you can use a temporary `redis` Docker image like this:
```

Comment:
Good idea to add this, but this seems to be lacking some context in this section. Can you try rephrasing this? :+1: